javascripts/discourse/services/excerpt-state.js

```diff
@@ -0,0 +1,21 @@
+import { tracked } from "@glimmer/tracking";
+import { action } from "@ember/object";
+import Service, { service } from "@ember/service";
+
+const EXCERPT_PREF_KEY = "showExcerpt";
+const EXCERPT_PREF = localStorage.getItem(EXCERPT_PREF_KEY);
```

Comment:
```suggestion
const EXCERPT_PREF = settings.show_toggle ? localStorage.getItem(EXCERPT_PREF_KEY) : "false";
```
I think we need something like this, so that if someone sets a preference in localstorage, and then the admin decides to remove the toggle, the localstorage value is ignored?